### PR TITLE
Added tutorial to README

### PR DESCRIPTION
**Pull request by Jaguar-515**
This tutorial could have been in Rojo's docs, however, LPG said on Discord that it should be in this repo's README because Rojo and this tool are separate tools from eachother.
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
 # rbxlx-to-rojo
 Tool to convert existing Roblox games into Rojo projects by reading their `rbxlx` place files.
 
+# Using rbxlx-to-rojo
+## Setup
+Before you can use rbxlx-to-rojo, you need to have Rojo 0.5.0 Alpha 12 to use the tool.
```
Comment (Kampfkarren) [inline README.md]: At *least* alpha 12.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+## Porting the game
+Let's assume we have a rbxlx file named "game", with a script in ServerScriptService.
+
+### Steps to port the game:
+1. Double-click on rbxlx-to-rojo on wherever you installed it.
+2. Select a place file.
+3. Select the path where you want to place it.
+
+If you followed the steps correctly, you should see something that looks like this:
+https://cdn.discordapp.com/attachments/609230525604954135/610155628786221117/unknown.png
```
Comment (Kampfkarren) [inline README.md]: Typo? You included this twice, but only one is an actual image.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+![](https://cdn.discordapp.com/attachments/609230525604954135/610155628786221117/unknown.png)
```
Comment (Kampfkarren) [inline README.md]: Can you save this image in a folder in the repo? So instead of this, it'd be like ``

**Comment by Jaguar-515** [comment]
I'll do the changes.... @Kampfkarren

File: README.md (hunk 1 above)
Comment (Jaguar-515) [reply README.md]: Fixed. Check the latest commit.

File: README.md (hunk 2 above)
Comment (Jaguar-515) [reply README.md]: Fixed. Check the latest commit.

File: README.md (hunk 3 above)
Comment (Jaguar-515) [reply README.md]: Fixed. Check the latest commit.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,24 @@
...
 Tool to convert existing Roblox games into Rojo projects by reading their `rbxlx` place files.
 
+# Using rbxlx-to-rojo
+## Setup
+Before you can use rbxlx-to-rojo, you need at least Rojo 0.5.0 Alpha 12 to use the tool.
+You also need a rbxlx place file that has a script. If there aren't any scripts in the rbxlx file, rbxlx-to-rojo will return an error.
+
+Download the latest release of rbxlx-to-rojo here: https://github.com/rojo-rbx/rbxlx-to-rojo/releases
+## Porting the game
+Let's assume we have a rbxlx file named "game", with a script in ServerScriptService.
```
Comment (Kampfkarren) [inline README.md]: Would rather show how to do this in the tutorial, rather than just assuming the user knows how to do it.

Comment (Jaguar-515) [reply README.md]: Lemme fix that.

Comment (Jaguar-515) [reply README.md]: Might have fixed that. Check the latest commit.

File: README.md [hunk 5]
```diff
@@ -1,5 +1,29 @@
...
+Before you can use rbxlx-to-rojo, you need at least Rojo 0.5.0 Alpha 12 to use the tool.
+You also need a rbxlx place file that has a script. If there aren't any scripts in the rbxlx file, rbxlx-to-rojo will return an error.
+
+Download the latest release of rbxlx-to-rojo here: https://github.com/rojo-rbx/rbxlx-to-rojo/releases
+## Porting the game
+Before you can port your game into Rojo projects, you need a .rbxlx file. If you have an existing game that isn't an .rbxlx file, Go to studio, click on any place, and then click on File -> Save to file as.
+
+Then, click on the drop-down menu where it says "Roblox Place Files(.rbxl).
+
+Click on the "Roblox XML Place Files(.rbxlx)", and then click "Save".
```
Comment (Kampfkarren) [inline README.md]:
```suggestion
Click on the "Roblox XML Place Files (.rbxlx)", and then click "Save".
```

File: README.md [hunk 6]
```diff
@@ -1,5 +1,29 @@
...
+# Using rbxlx-to-rojo
+## Setup
+Before you can use rbxlx-to-rojo, you need at least Rojo 0.5.0 Alpha 12 to use the tool.
+You also need a rbxlx place file that has a script. If there aren't any scripts in the rbxlx file, rbxlx-to-rojo will return an error.
+
+Download the latest release of rbxlx-to-rojo here: https://github.com/rojo-rbx/rbxlx-to-rojo/releases
+## Porting the game
+Before you can port your game into Rojo projects, you need a .rbxlx file. If you have an existing game that isn't an .rbxlx file, Go to studio, click on any place, and then click on File -> Save to file as.
+
+Then, click on the drop-down menu where it says "Roblox Place Files(.rbxl).
```
Comment (Kampfkarren) [inline README.md]:
```suggestion
Click on the drop-down menu where it says "Roblox Place Files (.rbxl)".
```

File: README.md [hunk 7]
```diff
@@ -1,5 +1,29 @@
...
 Tool to convert existing Roblox games into Rojo projects by reading their `rbxlx` place files.
 
+# Using rbxlx-to-rojo
+## Setup
+Before you can use rbxlx-to-rojo, you need at least Rojo 0.5.0 Alpha 12 to use the tool.
+You also need a rbxlx place file that has a script. If there aren't any scripts in the rbxlx file, rbxlx-to-rojo will return an error.
+
+Download the latest release of rbxlx-to-rojo here: https://github.com/rojo-rbx/rbxlx-to-rojo/releases
+## Porting the game
+Before you can port your game into Rojo projects, you need a .rbxlx file. If you have an existing game that isn't an .rbxlx file, Go to studio, click on any place, and then click on File -> Save to file as.
```
Comment (Kampfkarren) [inline README.md]: Please turn this into a list with bullet points, rather than the way it is now.

Comment (Jaguar-515) [reply README.md]: Might have fixed this, check the latest commit.

**Comment by Kampfkarren** [comment]
LGTM! Thanks!

**Pull request merged**